test/unit/httpexception.spec.js

```diff
@@ -29,4 +29,13 @@ describe('HttpException', function () {
     let error = new HttpException('Something went found')
     expect(error.status).to.equal(503)
   })
+
+  it('should use a fallback if captureStackTrace is not defined', function () {
+
+    // Monkey patch Error class to remove captureStackTrace() method
+    delete Error.captureStackTrace
+
+    let error = new HttpException(404, 'Page not found')
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>Missing semicolon.